### PR TITLE
GitHubAuthentication

### PR DESCRIPTION
**Pull request by MariaJose319**
image

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -18,3 +18,7 @@ jobs:
     - run: npm run build --if-present
     - name: Run tests
       run: npm test
```
Comment (holgiosalos) [inline .github/workflows/node.js.yml]: remove this step since is duplicated of the one you just added

Comment (MariaJose319) [reply .github/workflows/node.js.yml]: Thank you. Both lines?

Comment (MariaJose319) [reply .github/workflows/node.js.yml]: @holgiosalos  Can you see the second commit that I pushed?

**Review by veronicatofino — APPROVED** [review]
LGTM

**Comment by MariaJose319** [comment]
**I have a question about the last 2 workflow runs. I don't know why they didn't pass.**

image

image

**Comment by veronicatofino** [comment]
@MariaJose319 The test is failing by 401 - that is an error trying to authenticate. You can see the error here https://github.com/MariaJose319/workshop-api-testing-js-QAbootcamp/runs/7123724860?check_suite_focus=true#step:6:45  Did you configure the secret_token in the repo? If you like you can reopen this PR with the fix or in the next PR. I did not validate the Github Actions before the approval sorry

**Comment by MariaJose319** [comment]
> @MariaJose319 The test is failing by 401 - that is an error trying to authenticate. You can see the error here https://github.com/MariaJose319/workshop-api-testing-js-QAbootcamp/runs/7123724860?check_suite_focus=true#step:6:45 Did you configure the secret_token in the repo? If you like you can reopen this PR with the fix or in the next PR. I did not validate the Github Actions before the approval sorry

**In this part?**
image

**Comment by veronicatofino** [comment]
@MariaJose319 I talked you in slack to review

**Pull request merged**